dcos-log/journal/reader/read.go

```diff
@@ -103,9 +104,19 @@ func (r *Reader) Read(b []byte) (int, error) {
 			return 0, io.EOF
 		}
 
+		var (
+			c    uint64
+			cInt int
```

Comment:
this looks clumsy, but will be fixed once this is merged: https://github.com/coreos/go-systemd/pull/204